Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -96,6 +100,44 @@ public final class CharcoalViewController: UINavigationController {
         rootFilterViewController?.reloadFilters()
     }
 
+    public func removeFilter(_ filter: Filter) {
+        selectionStore.removeValues(for: filter)
+        handleFilterSelectionChange(from: .externalSearchFilterTag)
+    }
+
+    public func title(for filter: Filter) -> SelectionTitle? {
+        let titles = selectionStore.titles(for: filter)
+        guard titles.count == 1 else { return nil }
```

Comment:
I added this guard just in case the filter had more than one title. Filters with subfilters have multiple titles. Thought it might be confusing if you only got the first title back, if the filter happened to have multiple titles. 🤔

Reply:
Ah yes they could have multiple titles, that makes your code and my suggestion different 👍 😄